compiler.go

```diff
@@ -62,6 +62,11 @@ type Compiler struct {
 	// ExtractAnnotations tells whether schema annotations has to be extracted
 	// in compiled Schema or not.
 	ExtractAnnotations bool
+
+	// Loader overrides the document loading function for this compiler.
+	//
+	// If unset, the default global LoadURL function will be used.
+	Loader func(s string) (io.ReadCloser, error)
```

Comment:
change comment:

LoadURL loads the document at given URL.
If nil, package global LoadURL is used.

Reply:
Done